### ISSUE TITLE
Not in Maven central

### ISSUE THREAD

**Issue by davidgoate** [onfido/api-java-client#4]
I think it is the case that this library is not published to maven central. 

Adding to my pom.xml

```xml
<dependency>
    <groupId>com.onfido</groupId>
    <artifactId>onfido-java</artifactId>
    <version>1.5.0</version>
    <scope>compile</scope>
</dependency>
```

cannot resolve the dependency.  https://search.maven.org/search?q=g:com.onfido.api.client also yields no results. I see from https://mvnrepository.com/artifact/com.onfido.api.client/onfido-api-client/2.1.0 this the dependency is in jcenter (but under different group and artefact co-ordinates)

For many server-side (non android users who do not use jcenter as default) the central repo is defined as:

```xml
 <repository>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
      <id>central</id>
      <name>Central Repository</name>
      <url>https://repo.maven.apache.org/maven2</url>
    </repository>
```

I think it'd either be useful for devs to either:

1) publish to central too, which appear to be not as hard as it used to be, especially when using https://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central/ which suggests it's essentially a setting
2) Just document in the readme.md of this repository that you'd need to add something like:

```xml
<repository>
           <snapshots>
               <enabled>false</enabled>
           </snapshots>
           <id>jcenter</id>
           <name>bintray</name>
           <url>https://jcenter.bintray.com</url>
       </repository>
```

to your pom.xml.

**Comment by tsbatista** [comment]
Actually, I do not think this artifact is on any repository at all, it would be great if you were to publish it

**Comment by dimalusa** [comment]
:'( 
Same issue 
> Could not find com.onfido:onfido-java:4.0.0.

Should we use : https://mvnrepository.com/artifact/com.onfido.api.client/onfido-api-client/2.3.0 ?

**Comment by ajross** [comment]
Yep, it seems unnecessary to get everyone who wants to integrate to have to manage building, deploying, and checking this repo for updates.
